fearless/Common/Network/GitHubOperationFactory.swift

```diff
@@ -24,34 +24,45 @@ class GitHubOperationFactory: GitHubOperationFactoryProtocol {
                 return []
             }
 
-            var phishingItems: [PhishingItem] = []
-
             let addressFactory = SS58AddressFactory()
-            for (key, value) in json {
-                if let addresses = value as? [String] {
-                    for address in addresses {
-                        do {
-                            let typeRawValue = try addressFactory.type(fromAddress: address)
-
-                            guard let addressType = SNAddressType(rawValue: typeRawValue.uint8Value) else {
-                                continue
-                            }
-
-                            let accountId = try addressFactory.accountId(fromAddress: address,
-                                                                         type: addressType)
-
-                            let item = PhishingItem(source: key,
-                                                    publicKey: accountId.toHex())
-                            phishingItems.append(item)
+
+            let phishingItems = json.flatMap { (key, value) -> [PhishingItem] in
+                if let publicKeys = value as? [String] {
+                    let items = publicKeys.map {
+                        if let publicKey = self.getPublicKey(from: $0, using: addressFactory) {
+                            return publicKey
                         }
+                        return ""
```

Comment:
I think we can use compactMap instead of map and reduce the closure to just getPublicKey(...)